CMakeLists.txt

```diff
@@ -80,18 +79,23 @@ if (ENABLE_OPENMP)
 endif()
 if (ENABLE_CUDA)
   list(APPEND RAJA_PERFSUITE_DEPENDS cuda)
-endif() 
-if (ENABLE_HIP)
+endif()
+# Kokkos requires hipcc as the CMAKE_CXX_COMPILER for HIP AMD/VEGA GPU
+# platforms, whereas RAJAPerf Suite uses blt/CMake FindHIP to set HIP compiler 
+# Separate RAJAPerf Suite and Kokkos handling of HIP compilers
+if ((ENABLE_HIP) AND (NOT ENABLE_KOKKOS))
+#if (ENABLE_HIP)
```

Comment:
This line can be removed.

Reply:
DONE!  Apologies, I'm circling back around to your comments, and I'm just now seeing this - apologies for the oversight!